### ISSUE TITLE
Error: exit status 255

### ISSUE THREAD

**Issue by AndrewMac** [craftcms/nitro#454]
### Description
I've just started getting nitro errors when trying to install plugins.
Any idea what be might on? I've never run into this before and can't find anything about the error

### Steps to reproduce

```
nitro craft plugin/install whatever-plugin
connecting to spm.nitro
*** installing whatever-plugin
Error: exit status 255
```

### Additional info

- Nitro version: 2.0.9
- Docker version: 4.5.0 (74594), Mac Intel

**Comment by AndrewMac** [comment]
Please help! I'm running into this "error status 255" with all kinds of stuff now.
Can someone explain what it means, and what I should do? 
I'm tempted to uninstall and reinstall Nitro, but I don't know if I will lose my sites if I do that?

**Comment by AndrewMac** [comment]
Hi @jasonmccallister, and thanks for picking this up.

I'm afraid nothing happens when I ssh in. 
I get no feedback from any craft commands. 

For example:
```
/app $ php craft context
/app $
```

**Comment by AndrewMac** [comment]
I just tried creating a new Craft site, using 'nitro create ...' and I got an error during setup/welcome:
```
> @php craft setup/welcome

Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 7.4.28. in /app/vendor/composer/platform_check.php on line 24

Call Stack:
    0.0008     604632   1. {main}() /app/craft:0
    0.0022     607400   2. require('/app/bootstrap.php') /app/craft:8
    0.0027     608488   3. require_once('/app/vendor/autoload.php') /app/bootstrap.php:11
    0.0045     621960   4. ComposerAutoloaderInitb9e92cb94c5cb63a6c63424c2c7eaedb::getLoader() /app/vendor/autoload.php:7
    0.0063     626592   5. require('/app/vendor/composer/platform_check.php') /app/vendor/composer/autoload_real.php:25
    0.0063     627168   6. trigger_error($message = 'Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 7.4.28.', $error_type = 256) /app/vendor/composer/platform_check.php:24

Script @php craft setup/welcome handling the post-create-project-cmd event returned with error code 255
```
I was, however, able to nitro ssh and run install/welcome there, which worked.

Funny thing about the error above, though, is that I'm running PHP 8, not 7.

```
 ~/Sites/dev/craft> php -v
PHP 8.1.3 (cli) (built: Mar  1 2022 09:21:02) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.3, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.3, Copyright (c), by Zend Technologies
 ~/Sites/dev/craft> which php
/usr/local/bin/php
```
I don't really know what's going on, things have been working just fine for months, but suddenly ...

**Comment by AndrewMac** [comment]
Thanks so much for your help, but it looks like I might have been wasting your time. Sorry about that!

I think I might have solved the issue. I traced things back to a plugin called "Click and Collect". It's been installed for quite some time now, but for some reason it has just now started to cause problems. It's still even working on the live site, but strangely it's started causing all sorts of problems locally — first the nitro issues, and then (after I rebuilt the nitro environment for the local site, white screens for Craft itself). I've now removed it, and Nitro commands seem to have started working again.

I've also put in a feature request for Commerce to add that click-and-collect function ;-)
